.github/workflows/delivery-homebrew.yml

```diff
@@ -57,9 +58,9 @@ jobs:
       - run: cat homebrew-tap/Formula/pack.rb
       - name: Commit changes
         run: |
-          git config --global user.name "github-bot"
-          git config --global user.email "action@github.com"
-          
+          git config --global user.name "buildpack-bot"
+          git config --global user.email "cncf-buildpacks-maintainers@lists.cncf.io"
```

Comment:
Just to confirm - this is a valid email?

Reply:
Correct. It's what the project uses to manage accounts owned by the project and maintained by maintainers.